subxt/src/utils/bits.rs

```diff
@@ -0,0 +1,227 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+//! Generic `scale_bits` over `bitvec`-like `BitOrder` and `BitFormat` types.
+use codec::{
+    Compact,
+    Input,
+};
+use scale_bits::{
+    scale::format::{
+        Format,
+        OrderFormat,
+        StoreFormat,
+    },
+    Bits,
+};
+use std::marker::PhantomData;
+
+macro_rules! store {
+        ($ident: ident; $(($ty: ident, $wrapped: ty)),*) => {
+            /// Associates `bitvec::store::BitStore` trait with corresponding, type-erased `scale_bits::StoreFormat` enum.
+            ///
+            /// Used to decode bit sequences in runtime by providing `scale_bits::StoreFormat` using
```

Comment:
I don't really understand this sentence, can runtime be omitted? 

```suggestion
            /// Used to decode bit sequences by providing `scale_bits::StoreFormat` using
```

Reply:
I wanted to emphasize that StoreFormat is a dynamic configuration whereas the type parameters are compile-time but you're right, sticking runtime in here doesn't fit that much, since all the decoding is done in runtime anyway, regardless of the type of the decoding parameters. Will remove it, thanks!